eZ/Bundle/EzPublishCoreBundle/DependencyInjection/EzPublishCoreExtension.php

```diff
@@ -101,6 +101,9 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('security.yml');
         // Slots
         $loader->load('slot.yml');
+        if (interface_exists('FOS\JsRoutingBundle\Extractor\ExposedRoutesExtractorInterface')) {
```

Comment:
I was unsure of the possibility to get the accurate bundles list from the DIC extension... It's possible from a compiler pass, but the situation is different in the extension